Command/RequestPasswordResetHandler.php

```diff
@@ -46,10 +46,6 @@ public function handle(RequestPasswordDataTransferObject $dataTransferObject)
     {
         $user = $this->userRepositoryCollection->findUserByUserName($dataTransferObject->userName);
 
-        if ($user === null) {
-            throw new EntityNotFoundException();
```

Comment:
Nee, dat moet enkel staan in de method die effectief de exception throwt, daarna bubblet dat gewoon omhoog. In dit geval gaat `findUserByUsername()` die tag al bevatten